python3-requirements.json

```diff
@@ -0,0 +1,291 @@
+{
+    "name": "python3-requirements",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [          
+            {  "name": "lapack",
```

Comment:
this shouldn't  be in the `python3-requirements` since it is autogenerated and you added it by hand.

Reply:
This is observation is correct, I did add this by hand (lapack is a dependency of scipy). Never really considered that it may not be a great place in the auto-generated `python3-requirements` but you raise a good point as a future auto-generation  will save over this. Having it separate is much easier to maintain. Will fix!